### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.7][Eclipse][Wipro]:Keyboard focus is not visible for pane present in "Sign in with your phone or token device" dialog 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1527]
**Repro Steps:**

1. Open and Start Eclipse
2. Tab to Tools menu-> "Azure"-> "Azure Sign In"
3.  In "Azure Sign in" dialog, tab to and select "Interactive" radio button.
4. Tab to and select "Sign in" button.
5. In" Azure Login" Enter the "email or phone number" in the edit box and select Next button.
6. Tab to and select "Sign in with a user name and password instead" link.
7. Enter the required password and select "sign in" button.
8. Tab to and select "sign in with your phone or token device" link .
9. Tab to "Help and Support link" then press tab and observe the keyboard focus

**Expected Result:**
Keyboard focus should be visible for  pane present in  Azure Login Dialog

**Actual Result:**
Keyboard focus is not visible for pane present in Azure Login Dialog

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![keyboard focus is not visible for pane in azure login dialog1](https://user-images.githubusercontent.com/22956130/36192072-00b07f8a-1186-11e8-855b-023d0412b341.jpg)

**Comment by Eskibear** [comment]
under further investigation

**Comment by Eskibear** [comment]
duplicated with #1525

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.6.0 .
Issue still exists . 
Keyboard focus is not visible for pane present in Azure Login Dialog

**Comment by Eskibear** [comment]
it's the same as #1525 , tagged as external dependency

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749968
